### PR TITLE
Pipelines: Update Cekit path

### PR DESCRIPTION
**Pull request by radtriste**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO|RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

**Comment by winklerm** [comment]
The images are in place, the PR can be merged, thanks!

**Comment by radtriste** [comment]
Jenkins test this

**Comment by kie-ci1** [comment]
**PR job** `#418` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/418/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare for tests)<br/>Stage "Prepare for tests" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-images/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Mon Jan 23 07:48:09 EST 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Mon Jan 23 07:48:09 EST 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/418/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci1** [comment]
**PR job** `#419` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/419/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare for tests)<br/>Stage "Prepare for tests" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-images/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Mon Jan 23 07:55:58 EST 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Mon Jan 23 07:55:59 EST 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/419/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by radtriste** [comment]
jenkins retest this

**Comment by kie-ci1** [comment]
**PR job** `#420` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/420/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare for tests)<br/>Stage "Prepare for tests" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-images/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Mon Jan 23 08:05:42 EST 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Mon Jan 23 08:05:42 EST 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/420/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci1** [comment]
**PR job** `#421` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/421/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
ModuleNotFoundError: No module named 'ruamel'<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-images/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Mon Jan 23 08:13:15 EST 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Mon Jan 23 08:13:15 EST 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/421/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by radtriste** [comment]
successful build: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/blue/organizations/jenkins/KIE%2Fkogito%2Fmain%2Fpullrequest%2Fkogito-images/detail/kogito-images/425/pipeline/

File: .ci/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -201,9 +201,13 @@ String[] getImages() {
...
     }
-    return sh(script: "${listCmd} | tr '\\n' ','", returnStdout: true).trim().split(',')
+    return runPythonCommand("${listCmd} | tr '\\n' ','", true).trim().split(',')
 }
 
 boolean isProdCI() {
     return env.PROD_CI ? env.PROD_CI.toBoolean() : false
 }
+
+void runPythonCommand(String cmd, boolean stdout = false) {
```
Comment (spolti) [inline .ci/jenkins/Jenkinsfile:211]: is there a need to activate the virtual env for each command?

Comment (radtriste) [reply .ci/jenkins/Jenkinsfile:211]: yep. They mostly all use Python3 ... and specific package for python are also in the virtual env ...

**Review by spolti — APPROVED** [review]

**Pull request merged**